### PR TITLE
Code quality configuration for Codeac.io

### PR DESCRIPTION
**Pull request by michal-simon**
Hello Marcelo,
I briefly looked up your repo and saw a good amount of work you have done over here. Good job, man! 🥇

You recently also tried [Codeac.io](https://www.codeac.io?utm_source=marcelofmatos/phpnetmap) and you probably haven't had a good experience just because of lack of correct configuration and false positives.

I went through the biggest issues in your repository and added configuration to this pull-request to avoid irrelevant problems. I hope this will be helpful for you and find Codeac more useful after this.

Basically I ignored all external dependencies you have stored in the repository. This is just a workaround to help you get better results. The best practice would be to remove it from your repo completely and refer to it in `package.json` or `composer.json` instead.

Let me know if you need any more help.

Michal Šimon
CEO @ [Codeac.io](https://www.codeac.io?utm_source=marcelofmatos/phpnetmap)

PS: Keep doing open-source, please! We all appreciate your work a lot! :)

### CODE REVIEW

**Comment by michal-simon** [comment]
Hi Marcelo, how are you doing? I just wanted to ask if you had a chance to look at this PR. Is there anything blocking the merge? Happy to fix it. :)
Best, Michal
